utils/test_helper.ts

```diff
@@ -438,4 +438,42 @@ export class TestHelper {
             ...override,
         };
     }
+
+    public static getCloudCustomerMock(override: Partial<CloudCustomer> = {}): CloudCustomer {
+        return {
+            id: '',
+            billing_address: {
+                city: '',
+                state: '',
+                country: '',
+                postal_code: '',
+                line1: '',
+                line2: '',
+            },
+            company_address: {
+                city: '',
+                state: '',
+                country: '',
+                postal_code: '',
+                line1: '',
+                line2: '',
+            },
+            payment_method: {
+                type: '',
+                last_four: '',
+                exp_month: 0,
+                exp_year: 0,
+                card_brand: '',
+                name: '',
+            },
+            name: '',
+            email: '',
+            contact_first_name: '',
+            contact_last_name: '',
+            create_at: new Date(Date.now()).getUTCDate(),
```

Comment:
I'd recommend having create_at be `0` so that its in line with other mock functions in this helper, and so that it's easier to use if you want to use it to help generate an expected value that you will be using for checking deep equality.